src/api/migrations/20190620192200-create-user.js

```diff
@@ -0,0 +1,70 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
```

Comment:
don't you think it will be hard to check against them (UUID) cause they are too long, but do you have any specific reason on why we should use UUID